### PR TITLE
Choose PCNS Intake Form or Enquiry, Disclaimer pages

### PR DESCRIPTION
**Pull request by sanjaytkbabu**
# Description
Added choose pcns intake form or enquiry, disclaimer, cleanup, removed authentication restriction for Housing button

## Types of changes

New feature (non-breaking change which adds functionality) 

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

## Further comments
Ticket: [PADS-67](https://apps.nrs.gov.bc.ca/int/jira/browse/PADS-67)

### CODE REVIEW

File: frontend/src/views/StartView.vue [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+
+  .p-menuitem-text {
+    color: $app-link-text;
+  }
+
+  .p-menuitem {
+    color: $app-link-text;
+  }
+}
+</style>
```
Comment (kyle1morel) [inline frontend/src/views/StartView.vue:217]: This should still be scoped. Use the `:deep` annotator to access child classes. If you don't use scope the styles will bleed outside this component.

File: frontend/src/views/StartView.vue [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+
+<style scoped lang="scss">
+.p-breadcrumb {
+  border: none;
+  text-decoration: none;
+}
+
+.no-underline {
+  text-decoration: none;
+}
```
Comment (kyle1morel) [inline frontend/src/views/StartView.vue:155]: This already exists in PrimeFlex. https://primeflex.org/textdecoration

File: frontend/src/views/StartView.vue [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+
+.p-disclaimer {
```
Comment (kyle1morel) [inline frontend/src/views/StartView.vue:157]: Any particular reason we are using `p-`? This is not a PrimeVue class.

File: frontend/src/views/StartView.vue [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+  cursor: pointer;
+}
+
+.p-red {
+  color: red;
+}
+
+.span-underline {
+  text-decoration: underline;
+}
```
Comment (kyle1morel) [inline frontend/src/views/StartView.vue:180]: This already exists in PrimeFlex. https://primeflex.org/textdecoration

File: frontend/src/views/StartView.vue [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+    </div>
+    <div class="flex flex-wrap gap-3 justify-content-between mt-4">
+      <div>
+        <RadioButton
+          v-model="acceptDisclaimer"
+          value="accepted"
+          @click="flipAcceptDisclaimer"
+        />
+        <label
+          for="ingredient1"
```
Comment (kyle1morel) [inline frontend/src/views/StartView.vue:123]: ?

File: frontend/src/views/StartView.vue [hunk 6]
```diff
@@ -0,0 +1,217 @@
+<script setup lang="ts">
+import { ref } from 'vue';
+import { useRouter } from 'vue-router';
+import RadioButton from 'primevue/radiobutton';
```
Comment (kyle1morel) [inline frontend/src/views/StartView.vue:4]: Should be imported through `/lib/primevue`

File: frontend/src/views/StartView.vue [hunk 7]
```diff
@@ -0,0 +1,217 @@
...
+const acceptDisclaimer: Ref<string> = ref('');
+
+// Actions
+const router = useRouter();
+const handleStartApplication = () => {
+  if (enquiryOrIntake.value === 'intakeform') {
+    router.push({ name: RouteNames.INTAKE });
+  } else if (enquiryOrIntake.value === 'enquiry') {
+    router.push({ name: RouteNames.ENQUIRY });
+  }
```
Comment (kyle1morel) [inline frontend/src/views/StartView.vue:20]: Should be checking equivalence against the constants, not magic strings.
Can also be simplified (untested)
```ts
if ([RouteNames.ENQUIRY, RouteNames.INTAKE].includes(enquiryOrIntake.value)) {
    router.push({ name: enquiryOrIntake.value });
  }
```

File: frontend/src/views/StartView.vue [hunk 8]
```diff
@@ -0,0 +1,217 @@
...
+import { ref } from 'vue';
+import { useRouter } from 'vue-router';
+import RadioButton from 'primevue/radiobutton';
+import { Breadcrumb, Button } from '@/lib/primevue';
+import { RouteNames } from '@/utils/constants';
+import type { Ref } from 'vue';
+
+// State
+const enquiryOrIntake: Ref<string | null> = ref(null);
+const acceptDisclaimer: Ref<string> = ref('');
```
Comment (kyle1morel) [inline frontend/src/views/StartView.vue:11]: Consider just making this a boolean. You can then remove the function and on L120 just do `@click=acceptDisclaimer = !acceptDisclaimer`

File: frontend/src/views/StartView.vue [hunk 9]
```diff
@@ -0,0 +1,217 @@
...
+          for="ingredient1"
+          class="ml-2"
+        >
+          <span
+            class="span-agree"
+            @click="flipAcceptDisclaimer"
+          >
+            <b>Yes, I understand</b>
+          </span>
+          <span class="pl-1 span-red">*</span>
```
Comment (kyle1morel) [inline frontend/src/views/StartView.vue:132]: Was it a requirement to have the entire text clickable and not just the radio button?

File: frontend/src/views/StartView.vue [hunk 10]
```diff
@@ -0,0 +1,217 @@
...
+        relationship between you or your organization and Government of British Columbia. It may also be shared when
+        strictly necessary with partner agencies that are also subject to the provisions of the Act. Personal
+        information may be used by the Permitting Solutions Office for survey purposes. If you have any questions
+        regarding the use of this personal information, please contact Housing Authorizations at
+        <a href="mailto:Housing.Authorizations@gov.bc.ca">Housing.Authorizations@gov.bc.ca.</a>
+      </p>
+    </div>
+
+    <h4 class="h4-title mt-2">Before you begin</h4>
+    <div class="p-disclaimer">
```
Comment (kyle1morel) [inline frontend/src/views/StartView.vue:97]: Consider: https://primevue.org/card/

File: frontend/src/views/StartView.vue [hunk 11]
```diff
@@ -0,0 +1,217 @@
+<script setup lang="ts">
+import { ref } from 'vue';
+import { useRouter } from 'vue-router';
+import RadioButton from 'primevue/radiobutton';
+import { Breadcrumb, Button } from '@/lib/primevue';
+import { RouteNames } from '@/utils/constants';
+import type { Ref } from 'vue';
+
+// State
+const enquiryOrIntake: Ref<string | null> = ref(null);
```
Comment (kyle1morel) [inline frontend/src/views/StartView.vue:10]: Prefer `undefined` over `null`

File: frontend/src/views/StartView.vue [hunk 12]
```diff
@@ -0,0 +1,217 @@
...
+    class="container text-left pl-6 pr-6"
+  >
+    <h4 class="h4-title">Start a PermitConnect Housing Navigator Service application</h4>
+    <div class="p-disclaimer pt-2 pb-2">
+      <p>
+        <b>Apply for the PermitConnect Housing Navigator Service if you are working or getting started on a project</b>
+      </p>
+      <Button
+        class="btn-apply"
+        @click="enquiryOrIntake = 'intakeform'"
```
Comment (kyle1morel) [inline frontend/src/views/StartView.vue:59]: Use constants over magic strings.

File: frontend/src/views/StartView.vue [hunk 13]
```diff
@@ -0,0 +1,217 @@
...
+    </div>
+
+    <h4 class="mt-2 h4-title">Submit an enquiry</h4>
+    <div class="p-disclaimer pt-2 pb-2">
+      <p>
+        <b>Submit an enquiry if you have general questions, urgent requests, or want an update on your Application</b>
+      </p>
+      <Button
+        class="btn-apply"
+        @click="enquiryOrIntake = 'enquiry'"
```
Comment (kyle1morel) [inline frontend/src/views/StartView.vue:72]: Use constants over magic strings.

File: frontend/src/views/StartView.vue (hunk 3 above)
Comment (sanjaytkbabu) [reply frontend/src/views/StartView.vue:157]: Initially I was trying to apply that property to a <p>. I just changed it to div-disclaimer, as it is currently assigned to a <div>. does that work?

File: frontend/src/views/StartView.vue (hunk 8 above)
Comment (sanjaytkbabu) [reply frontend/src/views/StartView.vue:11]: This is great! thanks!

File: frontend/src/views/StartView.vue (hunk 7 above)
Comment (sanjaytkbabu) [reply frontend/src/views/StartView.vue:20]: Excellent suggestion!
I had to do it slightly differently as this kept giving me the following error:
'Argument of type 'string' is not assignable to parameter of type '"enquiry" | "intakeform"'

File: frontend/src/views/StartView.vue (hunk 6 above)
Comment (sanjaytkbabu) [reply frontend/src/views/StartView.vue:4]: exported RadioButton from frontend/src/lib/primevue/index.ts. Import works now. Thanks!

File: frontend/src/views/StartView.vue (hunk 1 above)
Comment (sanjaytkbabu) [reply frontend/src/views/StartView.vue:194]: :deep works! Thank you!

File: frontend/src/views/StartView.vue [hunk 14]
```diff
@@ -0,0 +1,228 @@
...
+  color: var(--text-color);
+}
+
+.p-red {
+  color: red;
+}
+
+.span-red {
+  color: red;
+}
```
Comment (kyle1morel) [inline frontend/src/views/StartView.vue:186]: Unnecessary to create multiple classes that do the same thing. A single `.red` will suffice. Alternatively https://primeflex.org/textcolor directly in the class annotator.

File: frontend/src/views/StartView.vue [hunk 15]
```diff
@@ -0,0 +1,228 @@
...
+  text-decoration: none !important;
+}
+
+.btn-apply {
+  width: 10rem;
+  justify-content: center;
+}
+.h4-title {
+  margin-bottom: 0;
+}
```
Comment (kyle1morel) [inline frontend/src/views/StartView.vue:198]: This is not being used. Also, PrimeFlex `mb-0` will accomplish this

File: frontend/src/views/StartView.vue [hunk 16]
```diff
@@ -0,0 +1,228 @@
...
+  padding: 0rem 1rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 0.5rem;
+}
+
+.span-agree {
+  color: var(--text-color);
+}
```
Comment (kyle1morel) [inline frontend/src/views/StartView.vue:178]: As this is only a colour override for the label, would suggest renaming to something like `.default-color`

File: frontend/src/views/StartView.vue [hunk 17]
```diff
@@ -0,0 +1,228 @@
...
+  </div>
+</template>
+
+<style scoped lang="scss">
+.p-breadcrumb {
+  border: none;
+  text-decoration: none;
+}
+
+.div-disclaimer {
```
Comment (kyle1morel) [inline frontend/src/views/StartView.vue:164]: `.disclaimer` is fine. Don't need to prefix with the element it resides in.

File: frontend/src/views/StartView.vue [hunk 18]
```diff
@@ -0,0 +1,228 @@
...
+  padding: 0px;
+  margin-bottom: 0.5rem;
+}
+:deep(.p-card-content) {
+  padding: 0px;
+}
+.custom-card {
+  border: none !important;
+  box-shadow: none !important;
+}
```
Comment (kyle1morel) [inline frontend/src/views/StartView.vue:227]: Should have a new line between classes.

File: frontend/src/views/StartView.vue [hunk 19]
```diff
@@ -0,0 +1,228 @@
...
+            </span>
+          </p>
+        </div>
+      </template>
+    </Card>
+    <div class="flex flex-wrap gap-3 justify-content-between mt-3">
+      <div>
+        <RadioButton
+          v-model="acceptDisclaimer"
+          value="accepted"
```
Comment (kyle1morel) [inline frontend/src/views/StartView.vue:136]: `value` needs to be updated as this is a boolean now.

File: frontend/src/views/StartView.vue [hunk 20]
```diff
@@ -0,0 +1,228 @@
+<script setup lang="ts">
+import { ref } from 'vue';
+import { useRouter } from 'vue-router';
+import { Breadcrumb, Button, Card, RadioButton } from '@/lib/primevue';
+import { RouteNames } from '@/utils/constants';
+import type { Ref } from 'vue';
```
Comment (kyle1morel) [inline frontend/src/views/StartView.vue:6]: Our spacing for imports is a bit specific. See FileUpload.vue for a small example.

```
3rd party imports

application imports

3rd party types
application types
```

Always alphabetized. The ordering here is correct, just missing the new lines.

File: frontend/src/views/StartView.vue (hunk 14 above)
Comment (sanjaytkbabu) [reply frontend/src/views/StartView.vue:186]: understood, updated with primeflex style.

File: frontend/src/views/StartView.vue (hunk 17 above)
Comment (sanjaytkbabu) [reply frontend/src/views/StartView.vue:164]: agreed

File: frontend/src/views/StartView.vue (hunk 20 above)
Comment (sanjaytkbabu) [reply frontend/src/views/StartView.vue:6]: Thank you!

**Pull request merged**